src/input/hotkey.c

```diff
@@ -362,6 +381,41 @@ void hotkey_enter(void)
     }
 }
 
+void hotkey_new_map(void)
+{
+    if(editor_is_active()) {
+        // TODO: here we need the size selection for triggering window_select_list_show?
+    } else {
+        building_construction_clear_type();
+        game_undo_disable();
+        game_state_reset_overlay();
+        window_main_menu_show(1);
+    }
+}
+
+void hotkey_load(void)
+{
+    if(editor_is_active()) {
+        window_editor_map_show();
+        window_file_dialog_show(FILE_TYPE_SCENARIO, FILE_DIALOG_LOAD);
+    } else {
+        building_construction_clear_type();
+        window_city_show();
+        window_file_dialog_show(FILE_TYPE_SAVED_GAME, FILE_DIALOG_LOAD);
+    }
+}
+
+void hotkey_save(void)
+{
+    if(editor_is_active()) {
+        window_editor_map_show();
+        window_file_dialog_show(FILE_TYPE_SCENARIO, FILE_DIALOG_SAVE);
+    } else {
+        window_city_show();
```

Comment:
This will also go to the city window when you are on the main menu, on the mission briefing or anywhere else.

Reply:
So should be solved with different check for "city window" being active (if possible, similar to editor map window check suggested above)?